### DISCUSSION TITLE
Weather widget broken again.

### DISCUSSION THREAD

**Discussion by mikal-s** [valinet/ExplorerPatcher#1683] [General]
Like happened a while ago weather widget now not working/loading again, assume as before windows update broke it.. get message has no access to internet... haha even now while here writing this.. so not a net fault!!  ideas anyone..??

**Comment by pyrates999** [comment]
Please post windows version and build.
Please post explorer patcher version.

Check what happens when using the console in explorer patcher.  Open up the properties application for explorer patcher, go to advanced, and click on enable console.  Then try to access the weather widget.

**Reply by mikal-s** [reply]
Here go pyates999, and thanks for reply,

ExplorerPatcher Ver 22621.1992.56.1
Windows 11 Pro, OS Build 10.0.22621.2070

Console:  Error in sub_1800135EC on RoGetActivationFactory.
[Updates] Configured update policy on this system: "Check for updates but let me choose whether to download and install them".
[Updates] Path to module: C:\WINDOWS\dxgi.dll
[Updates] Checking against hash "22622.1555.55.1.5758b678431cded2"
[Updates] Update URL: https://github.com/valinet/ExplorerPatcher/releases/latest/download/ep_setup.exe
[Updates] Unable to check for updates because the remote server is unavailable.
[Updates] Configured update policy on this system: "Check for updates but let me choose whether to download and install them".
[Updates] Path to module: C:\WINDOWS\dxgi.dll
[Updates] Checking against hash "22622.1555.55.1.5758b678431cded2"
[Updates] Update URL: https://github.com/valinet/ExplorerPatcher/releases/latest/download/ep_setup.exe
[Updates] Unable to check for updates because the remote server is unavailable.
[Updates] Configured update policy on this system: "Check for updates but let me choose whether to download and install them".
[Updates] Path to module: C:\WINDOWS\dxgi.dll
[Updates] Checking against hash "22622.1555.55.1.5758b678431cded2"
[Updates] Update URL: https://github.com/valinet/ExplorerPatcher/releases/latest/download/ep_setup.exe
[Updates] Unable to check for updates because the remote server is unavailable.

**Comment by pyrates999** [comment]
That is a preview update: https://support.microsoft.com/en-us/topic/july-26-2023-kb5028254-os-build-22621-2070-preview-66161ea6-c106-43e9-bef5-8410ccbd34e8

Uninstall it if you want to use explorer patcher and don't install preview updates in the future: https://github.com/valinet/ExplorerPatcher/wiki/Symbols

**Reply by mikal-s** [reply]
Was unaware I had previews installed (Until you pointed this out!) so checked and Windows has installed a few, on odd occasions.. and NO! did not click to install them.. (that I was aware of)
